### ISSUE TITLE
Any plans to support Orchestra's defn-spec macro?

### ISSUE THREAD

**Issue by matt3ho** [cursive-ide/cursive#1989]
Currently cursive does not recognize it as a defn form, even with the alt-enter menu it does not bring up the option to resolve it as a defn, though like Prismatic Schema, the form is a bit different from the standard defn.

Currently, any functions defined defn-spec ae not recognized as functions, leading to lots of unresolved symbols.

**Comment by sooheon** [comment]
I can resolve it as `defn` just fine, but because the syntax is so non-traditional, it doesn't really work. There is an extra arg after sym but before arg-list,

**Comment by jeaye** [comment]
I just received an email from someone this week asking how `defn-spec` can work with Cursive. That, combined with this ticket, shows there's definitely some interest.

Is there something I can do to help get `defn-spec` supported, short of changing the syntax of the macro?

**Comment by ionutbilica** [comment]
Yes please, it would help us. Resolving `defn-spec` as `defn` works for the function symbol but not for parameters.
